NDTensors/src/lib/SetParameters/src/get_parameter.jl

```diff
@@ -3,14 +3,15 @@
 
 Get a type parameter of the object `object` at the position `position`.
 """
-get_parameter(object, position::Position) = get_parameter(typeof(object), position)
+get_parameter(object::Type, position::Position) =
+  get_parameter(object, parameters(typeof(position))[1])
```

Comment:
I think we could define a shorthand for this through a conversion function, i.e. define `Base.Int(position::Position) = Int(parameters(typeof(position))[1])`.

Reply:
Also we could define `parameters(x) = parameters(typeof(x))` so then it could just be `Int(only(parameters(position)))`. We could also define a shorthand `parameter(x) = only(parameters(x))` as well for types that only have one type parameter.

Reply:
Okay so I am favoring `parameter` and `parameters` and phasing out the `get_parameter` function.  This leads to the question (and where I am working next). Do we want the `set_parameter` function to be replaced with a function like `parameter[s] = new_parameter` so it would be say
```
function parameter(type::Type, pos::Int, val)
  params = parameters(type)
  new_params = Base.setindex(params, val, pos)
  return set_parameters(type, new_params)
end
```

Reply:
I think we still want `set_parameter` (or another name if there is a better suggestion, but it should have `set` in the name).